mirroring/mirroring.go

```diff
@@ -136,46 +137,88 @@ func VerifySignedTreeHead() error {
 	return nil
 }
 
-func ComputeNewRootHash(oldRootHash string, logProof *models.ConsistencyProof) (string, error) {
-	newRootHash, err := hex.DecodeString(oldRootHash)
+func VerifyLogConsistency(oldSize int64, oldRootHash string) error {
+	rekorClient, err := client.GetRekorClient(rekorServerURL)
```

Comment:
Hmm, how is rekorServerURL resolved here?

Reply:
It is set as a constant on line 44 like this `const rekorServerURL = "https://api.sigstore.dev"`.
I was not sure if this is a good idea. Let me know if you have any other suggestion!

Reply:
Hmm, for now let's leave it as is, but I'd assume we want this to be overriddable (for the sake of testing and such). Can you make a ticket about making the api url a configuration parameter/setting?